README.md

```diff
@@ -1 +1,190 @@
-# Bookproject-ES6
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+    
+  <br/>
+
+  <h3><b>Tumaini Maganiko</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 AWESOME BOOKS <a name="about-project"></a>
+
+>This is my second microverse project I have built it with HTML and CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. CSS
+3. JAVASCRIPT
+4. Modules
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="/index.html">HTML</a></li>
+    <li><a href="/styles.css">CSS</a></li>
+    <li><a href="/index.js">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- Added index.html file
+- Added styles.css
+- Added index.js file
+- Used Modular JS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](COMING SOON)
+
```

Comment:
- [ ] Kindly make sure to remove the `Live Demo` section altogether from your README file, because the project is not deployed for now